internal/archive/archive.go

```diff
@@ -116,6 +147,57 @@ func (a *ubuntuArchive) Fetch(pkg string) (io.ReadCloser, error) {
 
 const ubuntuURL = "http://archive.ubuntu.com/ubuntu/"
```

Comment:
These three names feel a little off, they're all urls containing `ubuntu.com`, but the naming would imply to me that `ubuntuUrl` is a substring of the others. For consistency, could this be renamed to `ubuntuArchiveUrl`?

Reply:
The rules I have to follow say that I should not mix refactoring of existing code with other concerns. Let's make this in separate refactor PR?